doc/nrf/protocols/wifi/mem_requirements.rst

```diff
@@ -0,0 +1,50 @@
+.. _ug_wifi_mem_req:
+
+Memory requirements
+===================
+
+RAM and flash memory requirement values differ depending on the DK and the programmed sample.
```

Comment:
Code and RAM memory footprint requirements differ depending on the selected platform and the application example.